src/util/apiUtils.js

```diff
@@ -521,30 +521,32 @@ export const logApiResponse = request => ([response, body]) => {
 		statusCode
 	} = response;
 
+	const logger = request.server.app.logger;
 	// production logs will automatically be JSON-parsed in Stackdriver
-	const log = statusCode >= 400 && console.error ||
-		statusCode >= 300 && console.warn ||
-		console.log;
-
-	log(JSON.stringify({
-		message: `Incoming response ${method.toUpperCase()} ${pathname} ${response.statusCode}`,
-		type: 'response',
-		direction: 'in',
-		info: {
-			url: href,
-			query: (query || '').split('&').reduce((acc, keyval) => {
-				const [key, val] = keyval.split('=');
-				acc[key] = val;
-				return acc;
-			}, {}),
-			method,
-			id,
-			originRequestId: request.id,
-			statusCode: statusCode,
-			time: elapsedTime,
-			body: body.length > 256 ? `${body.substr(0, 256)}...`: body,
-		}
-	}));
+	const log = (statusCode >= 400 && logger.error ||
```

Comment:
I don't think so - `fatal` is for major 'system shutdown' kinds of errors, and `debug` is just for tracing code execution.